src/App.tsx

```diff
@@ -30,7 +30,7 @@ function App(): ReactElement {
 
   /**
    * Loads dependencies on initial render.
-   * Displays a spinner for the minimum of either: (a) the duration of the loading period or (b) 2 seconds.
+   * Displays a spinner for the maximum of either: (a) the duration of the loading period or (b) ~2 seconds.
```

Comment:
Should we maybe display a message telling the user to reload if the page is stuck somehow (load duration > some time)?

Reply:
Oh yeah, good point. Not sure what we would display but maybe a small picture or something to go with it.

I added a ticket here to tackle it: https://promopal.atlassian.net/browse/PP-99